README.md

```diff
@@ -984,6 +984,10 @@ in inheritance.
   is a hybrid of `Array`'s intuitive inter-operation facilities and
   `Hash`'s fast lookup.
 
+*  Prefer `map` over `collect`, `detect` over `find`, and `reduce` over
```

Comment:
fwiw `map`/`reduce` are also the terms used in JS, so frontend and ruby devs will be able to share the same terminology (would be true with `find` too but i think the ActiveRecord hazard trumps that)